src/classes/SRVC_ContactPrimaryLanguage_TEST.cls

```diff
@@ -0,0 +1,637 @@
+    @isTest 
+    private static void getContactLanguagesToUpdateAsNonPrimaryPositive(){
+        SRVC_ContactPrimaryLanguage.instance = new STUB_HasContactLanguagesToUpdate();
+        
+        Set<Id> testContactIdsSet = new Set<Id>();
+
+        for (Integer i = 0; i < 5; i++){
+            String testConId = UTIL_UnitTestData_TEST.getFakeId(Contact.SObjectType);
+            String testLangId = UTIL_UnitTestData_TEST.getFakeId(Language__c.SObjectType);
+
+            Contact_Language__c conLang = new Contact_Language__c(Id = UTIL_UnitTestData_TEST.getFakeId(Contact_Language__c.SObjectType),
+                                                                  Language__c = testLangId,
+                                                                  Contact__c = testConId,
+                                                                  Primary_Language__c = true);
+
+            testContactIdsSet.add(testConId);
+        }
+
+        Test.startTest();
+        List<Contact_Language__c> conLangsToSetNonPrimary = SRVC_ContactPrimaryLanguage.getInstance().getContactLanguagesToUpdateAsNonPrimary(testContactIdsSet);
+        Test.stopTest();
+
+        System.assertEquals(5, conLangsToSetNonPrimary.size(), 'There should be 5 Contact Languages to be updated to non-primary status.');
+    }                               
+
+    /**************************************************************************************************************************
+    * @description Test method to verify that the getContactLanguagesToUpdateAsNonPrimary method returns an empty list of 
+    * contact language records when an empty set of contact Ids is passed in as a parameter.
+    ***************************************************************************************************************************/
+    @isTest 
+    private static void getContactLanguagesToUpdateAsNonPrimaryEmpty(){
+        Test.startTest();
+        List<Contact_Language__c> conLangsToSetNonPrimary = SRVC_ContactPrimaryLanguage.getInstance().getContactLanguagesToUpdateAsNonPrimary(new Set<Id>());
+        Test.stopTest();
+
+        System.assertEquals(true, conLangsToSetNonPrimary.isEmpty(), 'There should be no Contact Languages to be updated to non-primary status.');
+    }
+
+    /**************************************************************************************************************************
+    * @description Test method to verify that the getDefaultFluencyFromSettings correctly returns the value set as the 
+    * Default_Contact_Language_Fluency__c.
+    ***************************************************************************************************************************/
+    @isTest 
+    private static void getDefaultFluencyFromSettingsPositive(){
+        Hierarchy_Settings__c hs = UTIL_CustomSettingsFacade.getSettingsForTests(new Hierarchy_Settings__c
```

Comment:
@m-kinkade This is a good example of where using getSettingsForTests is appropriate to test the veracity of that method.